Plugins/Flow.Launcher.Plugin.Folder/ContextMenuLoader.cs

```diff
@@ -89,7 +89,7 @@ public List<Result> LoadContextMenus(Result selectedResult)
                                 if (record.Type == ResultType.File)
                                     File.Delete(record.FullPath);
                                 else
-                                    Directory.Delete(record.FullPath);
+                                    Directory.Delete(record.FullPath, true);
```

Comment:
Perhaps adding an explanation like:

```suggestion
                                    Directory.Delete(record.FullPath, true); // `true` so it deletes recursively
```